### PR TITLE
Improves getting started documentation.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

## Summary
 - Improves installation instructions:
   - Bring pypi installation
 - Improves getting started page:
   - Provide info about bazel config

## Pendings:
 - [x] Bring maliput_examples repo to the getting started page
 - [x] Use multitab in the rest of the advanced/ getting started section

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

**Comment by francocipollone** [comment]
To better review changes here there are the sphinx's output: 
[docs.zip](https://github.com/maliput/maliput_documentation/files/13275085/docs.zip)
(Doxygen output and gallery videos aren't uploaded to reduce zip size.)

You need unzip and load `index.html` with your browser

File: docs/getting_started.rst [hunk 1]
```diff
@@ -135,6 +131,7 @@ Loading a RoadNetwork via Maliput Plugin Architecture
 
   We link against `maliput::api` and `maliput::plugin` for using the plugin interface.
   Note that we aren't linking against any maliput backend(`maliput_malidrive` in this case).
+  The plugin architecture is in charge of loading the backend in runtime.
```
Comment (agalbachicar) [inline docs/getting_started.rst:134]:
```suggestion
  The plugin architecture is in charge of loading the backend at runtime.
```

File: docs/getting_started.rst [hunk 2]
```diff
@@ -155,6 +152,37 @@ Loading a RoadNetwork via Maliput Plugin Architecture
...
+  const maliput::api::RoadGeometry* road_geometry = road_network->road_geometry();
+  const maliput::api::Lane* lane = road_geometry->ById.GetLane(maliput::api::LaneId{"1_0_1"});
+
+* `maliput::api::RoadGeometry::ToRoadPosition`_: Convert a inertial position to a road position.
+
+.. code-block:: cpp
+  :linenos:
+
+  const maliput::api::RoadGeometry* road_geometry = road_network->road_geometry();
+  maliput::api::RoadPositionResult road_position_result = road_geometry->ToRoadPosition(maliput::api::InertialPosition{10.0, 0.0, 0.0});;
```
Comment (agalbachicar) [inline docs/getting_started.rst:172]:
```suggestion
  const maliput::api::RoadPositionResult road_position_result = road_geometry->ToRoadPosition(maliput::api::InertialPosition{10.0, 0.0, 0.0});;
```

File: docs/getting_started.rst [hunk 3]
```diff
@@ -163,14 +191,14 @@ Maliput Python Interface
...
 
 .. note::
 
-  Check :ref:`installation_label` for setting up the ROS2 repositories and installing the packages via binaries.
+  Check :ref:`installation_label` for installing the packages via binaries.
 
-Once the dependencies are installed we can load a road network using the python interface. For doing so
-we are relying on the `maliput_py`_ package for the corresponding `maliput`_ bindings and the `maliput_malidrive`_ package as a `maliput`_ implementation.
+
+Whether it was installed via binaries (via ROS 2 or PyPI) or from source code, the python interface is available to be used.
```
Comment (agalbachicar) [inline docs/getting_started.rst:201]: I don't think it is super clear this section. I propose the  following write up:

```rst
It is required to have `maliput`_ and at least one backend, e.g. `maliput_malidrive`_ ,installed to use the python interface. It is possible to install them via ROS 2 build farm or PyPI.


.. note::

  Please refer to :ref:`installation_label` for installing the packages via binaries.
```

**Review by agalbachicar — CHANGES_REQUESTED** [review]
LGTM
Minor comments only.

File: docs/getting_started.rst (hunk 3 above)
Comment (francocipollone) [reply docs/getting_started.rst]: Indeed. thanks
Done.

File: docs/getting_started.rst (hunk 2 above)
Comment (francocipollone) [reply docs/getting_started.rst]: Done.

File: docs/getting_started.rst (hunk 1 above)
Comment (francocipollone) [reply docs/getting_started.rst]: Done.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**